docs/src/routes/docs/components/slider.svx

```diff
@@ -0,0 +1,100 @@
+---
+name: Slider
+---
+
+<script>
+  import { Slider } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+import { tick } from 'svelte';
+import Changelog from '../changelog.svx';
+</script>
+
+# Slider
+
+An slider for selecting a value or a range.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <div style="margin-left: 1.5em; height: 90%;">
+      <Slider
+        value={5}
+        min={1}
+        max={10}
+        step={1}
+        vertical
+        tooltips="active"
+        ticks={{mode: 'step', step: 1}}
+      />
+  </div>
+  </div>
+<div slot="source">
+
+```svelte
+  <Slider
+    value={5}
+    min={1}
+    max={10}
+    step={1}
+    vertical
+    tooltips="active"
+    ticks={{mode: 'step', step: 1}}
+  />
+```
+</Showcase>
+
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`vertical`** | `false` | `boolean` | To position the slider vertically with values from top to bottom. |
+| **`rectangularHandle`** | `false` | `boolean` | To have rectangular handles instead of the default round ones. |
+| **`disabled`** | `false` | `boolean` | Will disabled the slider in tones of grey, with cursor not-allowed on slider controls. | 
+
+### Class Properties
+
+Falsy values passed to classes will be disregarded.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`min`** | | `number` | The min value of the slider. |
+| **`max`** | | `number` | The max value of the slider. |
+| **`value`** | | `number | [number, number]` | Provide two numbers to create a range slider. |
```

Comment:
not sure what you want to but when there is no default and the prop is required, I think for readers of the docs it would be helpful if it had **required** in this column